src/App.js

```diff
@@ -9,6 +9,7 @@ class App extends Component {
     this.state = {
       movies: [],
       chosenMovie: null,
+      showingMain: true,
```

Comment:
@k-atwhite Do you need to have this additional property in state or can you revert back to the main page if chosenMovie is null? 
You only have a few properties in state here, but in further projects and in your working life it will be less messy if you can refactor your state properties to only what you absolutely need :)

Reply:
I 100% agree with Jessica here